tool/jt.rb

```diff
@@ -404,7 +404,8 @@ def help
           parser                                     build the parser
           options                                    build the options
           cexts                                      build only the C extensions (part of "jt build")
-          native [sulong] [extra mx image options]   build a native image of TruffleRuby, optionally with Sulong
+          native [--no-sulong] [extra mx image options]  
+                                                     build a native image of TruffleRuby, optionally with Sulong
```

Comment:
Why changing the default? For pre-init, etc, it's pointless to build with Sulong and just takes more time/memory.
The option format is indeed weird, so I'd be fine to change `jt build native sulong` to `jt build native --sulong`.

Reply:
The default should produce fully working ruby executable. I think that is what is expected, no surprise is the motivation not compilation time. If you care about time and you don't have a use for sulong you can always use `--no-sulong` option.

Reply:
Arguably without Sulong we have a ruby executable working for many things, except C extensions. It's anyway our own tool (so far at least), so I don't think there are expectations for it to be a "complete ruby SVM image like in GraalVM".
I see it as a tool to debug and experiment with the Ruby image, and Sulong is rarely needed for my debugging/experiments.

It might be useful to get new data on image build times with recent SVM changes, maybe the difference isn't that much.

Reply:
Gem install does not work without Sulong, I see that as quite major problem. All our builds require Sulong by default, I think it's quite preferable to have it unified and have Sulong everywhere. I expect that all our builds are complete as in GraalVM as much as possible, I hate having to thing about what is missing where or why a particular build differs. Yes, it's a command for building a standalone native image locally but why would we complicate our lives with making it different then other builds, especially if it's suppose to be for debugging. Over time I think it should converge to be identical with the standalone distribution. Who'll get the data? I won't have time until next week.

Reply:
The gem case is a good point. Since I need this PR in to avoid conflicts, let's take `--no-sulong` for now.